docs/_docs/versions.md

```diff
@@ -7,112 +7,161 @@ redirect_from: /docs/
+## Which version can users access?
 
-Or if you prefer to export your app for the CLI environment, learn how to do that [here](./export) and skip the cloning portion of this guide.
+A Zapier integration can be either _Private_, _Beta_, or _Public_. Private integrations require an [invitation](https://platform.zapier.com/docs/testing#how-to-invite-others-to-test-new-integrations) for users to access it. Public and Beta integrations can be seen by Zapier users on [the public Apps page](https://zapier.com/apps). Each integration can have many versions, but only one version can be public at a time, while the rest remain private. 
 
-## Clone Your Integration
+Newly published integrations will feature a _Beta_ tag when first published. While your integration is in Beta, we’ll monitor how it’s performing. When your integration reaches 50 users, our team will get in touch to invite you to start the [official partner program](https://zapier.com/platform/partner-program) launch process.
 
-![Clone Zapier integration](https://cdn.zapier.com/storage/photos/dca2130ce5dddca928519ad60130d35a.png)
+To review your integration’s versions, open it in the Developer Platform and navigate to Manage then Versions. This page shows a list of all versions of the integration, along with status and the number of users of each.
 
-When creating a new version of your Zapier integration, you never need to start over. Instead, to make a new version, start by cloning an existing version. That lets you start with an exact copy of your integration where you can make the changes and additions you need.
+![The Versions page in the Zapier Developer Platform](https://cdn.zappy.app/220a5f0055ab88b41ec83bfa953db20b.png)
 
-To clone a version, click the gear icon in the _Versions_ page, and select _Clone_. In the dialog, select the version to clone from (usually the most recent version, unless you want to roll back recent changes). Then choose what type of version you want to create:
+> **Note:** In the Zapier CLI, you can also run `zapier versions` to see the information in your terminal.
 
-- **Patch**: A `x.x.1` version, typically used to fix bugs and issues in existing integrations
-- **Minor**: A `x.1.x` version, typically to add new minor features to existing integrations
-- **Major**: A `2.x.x` version to launch a full new version of your integration with new triggers and or actions
+For Public apps, which are searchable in the Zap Editor/App Directory, a user that selects your integration in the Zap Editor will be using the current Public version by default. Optionally, you can use the Sharing page to send users an invitation to a specific private version, which will allow them to select it in the Zap Editor. This is also how all users of Private integrations are added.
```

Comment:
Do we maybe also wanna link the how to invite doc here https://platform.zapier.com/docs/testing#how-to-invite-others-to-test-new-integrations